packages/manager/apps/pci-gateway/src/pages/edit/EditGateway.page.tsx

```diff
@@ -59,9 +61,19 @@ export default function EditGatewayPage(): JSX.Element {
 
   const sizes = useData(projectId);
 
+  const regionName = searchParams.get('region');
```

Comment:
Could return `null` so it should be handled before the `find` execution to make typing explicit and improve performance